packages/remark-shaku-code-annotate/src/index.ts

```diff
@@ -286,11 +292,16 @@ function parseComment(line: IThemedToken[]): null | {
   };
 }
 function parseLines(lines: IThemedToken[][]) {
+  let isShaKuLine = false;
   return lines.map((line) => {
     const parsedComment = parseComment(line);
     if (parsedComment != null) {
       const { body, offset } = parsedComment;
       const shakuLine = parseLine(body);
+      if ((/\/\*/.test(line[0].content)) && (shakuLine?.type === "DirectiveCallout" || shakuLine?.type === "DirectiveUnderline" || shakuLine?.type === "AnnotationLine")) {
+        isShaKuLine = true;
```

Comment:
this assumes comment block all starts with `/*`? which is not solid
also all shaku lines should be supported